qiskit/circuit/quantumcircuit.py

```diff
@@ -4683,6 +4684,199 @@ def mcp(
             copy=False,
         )
 
+    def mcrx(
+        self,
+        theta: ParameterValueType,
+        q_controls: QuantumRegister | list[Qubit],
+        q_target: Qubit,
+        use_basis_gates: bool = False,
+    ):
+        """
+        Apply Multiple-Controlled X rotation gate
+
+        Args:
+            theta: angle theta
+            q_controls: The list of control qubits
+            q_target: The target qubit
+            use_basis_gates: use p, u, cx
+        """
+        # pylint: disable=cyclic-import
+        from .library.standard_gates.rx import RXGate
+        from qiskit.synthesis.multi_controlled import (
+            _apply_cu,
+            _apply_mcu_graycode,
+            _mcsu2_real_diagonal,
+        )
+
+        control_qubits = self._qbit_argument_conversion(q_controls)
+        target_qubit = self._qbit_argument_conversion(q_target)
+        if len(target_qubit) != 1:
+            raise QiskitError("The mcrx gate needs a single qubit as target.")
+        all_qubits = control_qubits + target_qubit
+        target_qubit = target_qubit[0]
+        self._check_dups(all_qubits)
+
+        n_c = len(control_qubits)
+        if n_c == 1:  # cu
```

Comment:
With @alexanderivrii's PR on the annotated gate plugins, could we change this method in future to just append an annotated Pauli rotation? It would be nice if we can move this logic to the plugin so it doesn't need to be duplicated for the annotated rotation.

Reply:
Indeed. I think this could be done in a follow-up PR.